.gitignore

```diff
@@ -57,6 +57,7 @@ docs/_build/
 #*#
 
 #venv
```

Comment:
Should the comment be removed since venv has been added?

Reply:
I think that comment is just trying to say this is a section for virtualenv.

Reply:
Ah, that makes sense. Thank you!